pytest_fixtures.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--test_db_url',
```

Comment:
I'll replace underscores with dashes in the sqlalchemy-filters as well 👍 

The idea of having `db-dialect`, `db-driver` and `test-db-uri` as separate options was that you could replace some of them, that are more likely to be different (authority...), without having to provide the rest, like the scheme (dialect + driver). I started having MySQL as the default database and this was very convenient, since the default DB host was `localhost`, but I was using a Docker container, so I could leave the scheme as it was and just provide the server and port. The same thing happens if you want to use the default server and port (`localhost` and `3306`) but use PostgreSQL instead of SQLite: you just have to provide the scheme for it (you can also do it using the Makefile) and not the whole URI.

The Makefile accepts several variables for that (like `DB_DIALECT` and `DB_DRIVER`) that are just passed to pytest,  and the SQLAlchemy URI is build in our Python code (conftest.py https://github.com/Overseas-Student-Living/sqlalchemy-filters/blob/master/test/conftest.py#L41), rather than having the logic in the Makefile.